CHANGELOG.md

```diff
@@ -3,6 +3,33 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [2.2007.2] - 2020-08-26
+
+### Fixed
+  - compaction: Use separate compactors for L0, L1 (#1466)
+  - Rework Block and Index cache (#1473)
+  - Add IsClosed method (#1478)
+
+### New APIs
+- Badger.DB
+  - BlockCacheMetrics (#1473)
+  - IndexCacheMetrics (#1473)
```

Comment:
Should BfCacheMetrics, DataCacheMetrics be part of removed APIs?

Reply:
Yes. They should be. Nice catch. Fixed it.